pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1735,7 +1735,7 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(len(causes)).To(Equal(0))
 		})
 
-		It("should reject SRIOV interface when feature gate is disabled", func() {
+		It("should accept SRIOV interface when feature gate is disabled", func() {
```

Comment:
In my mind, we have no code for SR-IOV in the admitter so it's arguable if we need to have any test cases for sriov here. But I am ok either way.